proto/substrait/algebra.proto

```diff
@@ -843,6 +857,12 @@ message Expression {
     // Optional; defaults to the end of the partition.
     Bound upper_bound = 4;
 
+    // Defines the window frame: ROWS, RANGE
+    WindowFrame window_frame = 12;
+
+    // Defines window column name for current window function.
+    string window_column_name = 13;
```

Comment:
What is this?

Reply:
This is only to pass the column name with window operator. And we can also store it in AdvancedExtension node.

Reply:
Substrait doesn't normally name columns in the middle of a plan so I think we can remove this.

Reply:
Yes. I already removed it.